_posts/2015-07-21-the-epages-selenium-framework.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "The ePages Selenium Framework"
+date: "2015-07-21 17:06:00"
+icon: ship
+tags: selenium, test, continuous delivery
```

Comment:
Please use this pattern to display the tags: ["yuzuiz", "asdsa asds", "sadsad asdsa"]